pkgs/servers/sql/postgresql/ext/apache_datasketches.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, lib, fetchFromGitHub, gcc, postgresql, boost182, nixosTests }:
+
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache_datasketches";
```

Comment:
The name in the derivation doesn't match the title in the PR, this is why tests are not triggered. This needs to be fixed as well.